### ISSUE TITLE
Compiling new SDK

### ISSUE THREAD

**Issue by wbadry** [fernandozuher/naoqi_webots#1]
Hello Fernando,
Were you able to get to know how to replace the old SDK with the recent one in order to make it work on Linux? I guess for motion and vision, both V5 and V6 are compatible. I hold no clue about the migration of the code to V6.

**Comment by fernandozuher** [comment]
Hello, @wbadry!

First of all: I'm sorry for this delayed response. I read your message just a few minutes ago. I hadn't received any e-mail from GitHub about this opened issue.

I made contact with SoftBank Robotics, RobotLab and a Spanish reseller of NAO.
I received some interesting suggestions about how I might solve the current issue.

In spite of that I had stopped to go on because of limited free time.
Although it is not ideal, Choreographe was being "ok" for what I was experimenting with NAO.
But things have changed one week ago when I needed a bit more complexity in Choreographe: I faced weird bugs in it which are probably related to my Ubuntu 18.04, whereas it was developed to work on Ubuntu 16.04.

I am supposed to be less busy on next month. So I can go on on that. :-)
Thanks for your interest.
